grommet/grommet
# Fix DateInput assuming a range on a non range format

#### What does this PR do?
It makes the DateInput recognize a date format `mm-dd-yyyy`as a single date and not range

#### Where should the reviewer start?
The Reviewer shoud start

#### What testing has been done on this PR?
I tested it on the storybook and also wrote tests for it.

#### How should this be manually tested?
Try to create a DateInput with a `mm-dd-yyyy` and check that its working as a range 

#### Do Jest tests follow these best practices?


#### Any background context you want to provide?

#### What are the relevant issues?
https://github.com/grommet/grommet/issues/5922

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
No

#### Should this PR be mentioned in the release notes?
Yes

#### Is this change backwards compatible or is it a breaking change?
It could be a breaking change if anyone is relying on this buggy behaviour.
